### ISSUE TITLE
The input pages for follow ups doesn't seem to be checking input for errors. 

### ISSUE THREAD

**Issue by sthomas313-gatech** [Co-Epi/app-ios#275]
**Describe the bug**
NOTE: I am running on a simulator rather than a physical device. 

**To Reproduce**
1. Go to 'CoEpi' App
2. Click on 'Symptom Report'
3. Click on any of the symptoms to prompt a followup.
4. Enter non-numeric values.
5. Click "Next" and it should go through without error checking.

**Expected behavior**
It should either return error to user on screen and explain input was invalid or something else of that nature.

**Screenshots**
![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-06-17 at 09 57 00](https://user-images.githubusercontent.com/42740535/84908386-65e58880-b082-11ea-8b04-5e20a3946059.png)
![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-06-17 at 09 56 15](https://user-images.githubusercontent.com/42740535/84908388-667e1f00-b082-11ea-94c1-c19b6cb35142.png)

**Comment by i-schuetz** [comment]
For completeness, how did you enter the text (at least the keyboard should be numeric)?
@Sam-gtri https://github.com/Co-Epi/app-ios/issues/191

**Comment by ivnsch** [comment]
For completeness, how did you enter the text (at least the keyboard should be numeric)?
@astewart43-gtri #191

**Comment by ivnsch** [comment]
Ah, simulator. You probably entered them using the computer's keyboard. This is a corner case. Not sure whether it should be prioritized for 0.3. @danamlewis 

For the record, the invalid input is discarded (i.e. handled as if the user didn't enter anything) and shows an error message in the logs.

**Comment by sthomas313-gatech** [comment]
Yes the computer keyboard allows me to enter anything, including emojis into the field. But yes, I understand this a corner case.

**Comment by sthomas313-gatech** [comment]
While checking out accessibility elements, one of the things I found was : Settings  > Accessibility > Keyboards, tap Full Keyboard Access, then turn on Full Keyboard Access. This will allow for people who is external keyboards to have additional inputs like I had on the simulator.

**Comment by ivnsch** [comment]
We definitely need to add validation. It's not something I personally would prioritize right now, though. Maybe as a refinement for 0.6 or so.

**Comment by ivnsch** [comment]
I'll set assign a hypothetic 0.6 milestone to this, just to signalize that we want to solve it "at some point in the future".
